docs/content/users/providers/lagoon.md

```diff
@@ -0,0 +1,18 @@
+# Amazee Lagoon Integration
+
+DDEV provides integration with the [Amazee Lagoon](https://lagoon.sh/), allowing Lagoon users to quickly download database and files from a Lagoon project to the local DDEV project.
+
+## Amazee Lagoon Per-Project Configuration
+
+1. Check out the Lagoon project and configure it with [`ddev config`](../usage/commands.md#config). You’ll want to use [`ddev start`](../usage/commands.md#start) and make sure the basic functionality is working.
+2. Configure an SSH key at on the [Amazee Cloud Dashboard](https://dashboard.amazeeio.cloud/settings).
+3. Add LAGOON_PROJECT and LAGOON_ENVIRONMENT variables to your project in 'web_environment' or a '.ddev/.env'. For example, `ddev config --web-environment-add="LAGOON_PROJECT=<project-name> LAGOON_ENVIRONMENT=<environment-name>"`.
+4. `ddev auth ssh` if you haven't already done so. This will make your SSH key available in the web container of your project.
```

Comment:
Run commands, edit for brevity. (No harm in running again, and someone will presumably know if they’ve run the command already—and maybe to run it again if their SSH key is not available in the web container.)

```suggestion
4. Run `ddev auth ssh` to make your SSH key available in the project’s web container.
```